src/python/foglamp/core/server_daemon.py

```diff
@@ -75,38 +83,54 @@ def start():
             _start_server()
 
 
-def stop():
-    """Stops the FogLAMP process if it is running"""
+class StopProcessFailed(Exception):
+    pass
```

Comment:
great addition 👍 , eventually we will need to have exceptions.py module for each package for our definitions